cmd/api/auth/authz.go

```diff
@@ -4,49 +4,50 @@
 package auth
 
 import (
-	"log"
+	"fmt"
 	"net/http"
 
 	"github.com/gin-gonic/gin"
+	authnv1 "k8s.io/api/authentication/v1"
```

Comment:
```suggestion
	apiAuthn "k8s.io/api/authentication/v1"
```

I think you can drop the "v1" from this abbreviation. I doubt we will be using v1 and v2 at the same time. But, you should add "api" to make it clearer what package this is.

Reply:
i think that we should keep the `v1` in the name since the standard way of referring to k8s apis is to include the version number

Reply:
Changed to `apiAuthnv1`. See previous thread for more info.